src/petals/client/inference_session.py

```diff
@@ -103,8 +104,11 @@ def step(
             assert len(hypo_ids) == len(new_hidden_states)
             assert hypo_ids.dtype == torch.int64
 
+        if attention_mask is None:
+            attention_mask = DUMMY
+
```

Comment:
If attention mask is None, inputs = (new_hidden_states, prompts, hypo_ids)
# TODO add a dummy attention mask here in 1.2.0